### ISSUE TITLE
False positive when list is extended via augmented assign operator

### ISSUE THREAD

**Issue by rogalski** [PyCQA/pylint#1433]
### Steps to reproduce
```
# pylint: disable=missing-docstring,invalid-name
import xml.etree.ElementTree as ET

xml_text = "<root><a><name>A</name></a><b><name>B</name></b></root>"
tree = ET.fromstring(xml_text)
xml_nodes = []
for search_path in ['./a', './b']:
    xml_nodes += tree.findall(search_path)

for node in xml_nodes:
    text = node.find('name').text
    print(text)
```

### Current behavior
```
E: 10,16: Instance of 'list' has no 'find' member (no-member)
```

### Expected behavior
No violations reported.

### pylint --version output
```
(pyve) C:\Users\lrogalsk\Repo\autobench_next-gpva>pylint --version
No config file found, using default configuration
pylint 1.7.1,
astroid 1.5.2
Python 2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 20:42:59) [MSC v.1500 32 bit (Intel)]
```

**Comment by mbyrnepr2** [comment]
I can't reproduce with the latest Pylint; can be closed?

**Comment by DanielNoord** [comment]
Thanks @mbyrnepr2! Closing this :)
